Makefile

```diff
@@ -21,3 +21,7 @@ docs:
 
 clean:
 	rm -rf docs/build
+
+format:
+	poetry run yapf -r -i probably/ tests/
+	poetry run isort probably/ tests/
```

Comment:
`isort` is not a dev-dependency, does this actually work?

Reply:
It runs on my machine... Maybe some other package depends on `isort`? In any case, if this should stay here then we should probably add it to the dependencies